setup.py

```diff
@@ -53,6 +53,6 @@
     setup_requires=["cffi"],
     ext_package="metaswitch.common",
     cffi_modules=["cffi_build.py:ffi"],
-    install_requires=["py-bcrypt", "pycrypto", "pyzmq", "cffi"],
+    install_requires=["py-bcrypt", "pycrypto", "pyzmq", "cffi", "monotonic"],
```

Comment:
Worth pinning the version of monotonic used? Do you need to update https://github.com/Metaswitch/clearwater-docs/blob/master/README.txt?